File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/DocumentFieldType.scala

package org.openapitools.client.model


case class DocumentFieldType (
    /* Descriptive name of this field type. */
    _label: String,
    _numberValueTypeMetadata: Option[DocumentFieldTypeNumberValueTypeMetadata],
    /* The kind of value that can be submitted for this fieldType. It should be either ValueType_Number, ValueType_String, or ValueType_Photo. */
    _valueType: String
)
object DocumentFieldType {
    def toStringBody(var_label: Object, var_numberValueTypeMetadata: Object, var_valueType: Object) =
        s"""
        | {
        | "label":$var_label,"numberValueTypeMetadata":$var_numberValueTypeMetadata,"valueType":$var_valueType
        | }
        """.stripMargin
}

File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/DocumentFieldType.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.DocumentFieldTypeNumberValueTypeMetadata
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param label Descriptive name of this field type.
 * @param numberValueTypeMetadata 
 * @param valueType The kind of value that can be submitted for this fieldType. It should be either ValueType_Number, ValueType_String, or ValueType_Photo.
 */
data class DocumentFieldType (

        @get:NotNull 
        @JsonProperty("label") val label: String,

        @get:NotNull 
        @JsonProperty("valueType") val valueType: String,

        @JsonProperty("numberValueTypeMetadata") val numberValueTypeMetadata: DocumentFieldTypeNumberValueTypeMetadata? = null
) {

}

